app/components/file-details/api-scan/captured-apis/index.ts

```diff
@@ -302,7 +334,13 @@ export default class FileDetailsApiScanCapturedApisComponent extends Component<F
       this.fetchAndSetSelectedApiCount.perform();
     } catch (error) {
       const err = error as AdapterError;
-      this.notify.error(err.toString());
+
```

Comment:
Can we maintain same error messages as previously done? Update for the other catch blocks in this file.